### PR TITLE
airbrake_deployment: Add version param

### PR DESCRIPTION
**Pull request by leedm777**
##### SUMMARY
The aibrake v4 API allows for distinct `version` and `revision` params.
The `revision` param is meant to indicate a revision from the version
control system (such as a Git hash), whereas the `version` param is
meant to be a version number (such as 1.2.3). This is especially
noticeable in the Airbrake UI where revisions are truncated to 7
characters, and used to build GitHub style diff links (such as
`https://github.com/ansible-collections/community.general/compare/689a25edcf03a3b8513faf347a957d2ad443c124...e54dd3a01f2c421b558ef33b5f79db936e2dcf15`).

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
airbrake_deployment

##### ADDITIONAL INFORMATION

none

```paste below

```

### CODE REVIEW

File: changelogs/fragments/airbrake_deployment_add_version.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - "airbrake_deployment - Add ``version`` param; clarified docs on ``revision`` param"
```
Comment (felixfontein) [inline changelogs/fragments/airbrake_deployment_add_version.yml:3]:
```suggestion
  - "airbrake_deployment - add ``version`` param; clarified docs on ``revision`` param (https://github.com/ansible-collections/community.general/pull/583)."
```

File: plugins/modules/monitoring/airbrake_deployment.py [hunk 2]
```diff
@@ -48,7 +48,12 @@
     type: str
   revision:
     description:
-      - A hash, number, tag, or other identifier showing what revision was deployed
+      - A hash, number, tag, or other identifier showing what revision from version control was deployed
+    required: false
+    type: str
+  version:
+    description:
+      - A string identifying what version was deployed
```
Comment (felixfontein) [inline plugins/modules/monitoring/airbrake_deployment.py:56]:
```suggestion
      - A string identifying what version was deployed
    version_added: 1.0.0
```

File: plugins/modules/monitoring/airbrake_deployment.py [hunk 3]
```diff
@@ -103,6 +117,7 @@ def main():
             user=dict(required=False),
             repo=dict(required=False),
             revision=dict(required=False),
+            version=dict(required=False),
```
Comment (felixfontein) [inline plugins/modules/monitoring/airbrake_deployment.py:120]:
```suggestion
            version=dict(required=False, type='str'),
```

Comment (leedm777) [reply plugins/modules/monitoring/airbrake_deployment.py:120]: Should types be added to the other params?

File: plugins/modules/monitoring/airbrake_deployment.py (hunk 2 above)
Comment (leedm777) [reply plugins/modules/monitoring/airbrake_deployment.py:56]: I added `version_added`, but put it at the end of the `version` block, and quoted, to match the other `version_added` fields earlier in this file.

Thanks!

Comment (felixfontein) [reply plugins/modules/monitoring/airbrake_deployment.py:56]: Perfect :) With the suggestion feature it was easier to place the suggestion here, the place you put it is even better.

File: plugins/modules/monitoring/airbrake_deployment.py (hunk 3 above)
Comment (felixfontein) [reply plugins/modules/monitoring/airbrake_deployment.py:121]: Yes please. But feel free to do it in another PR.

**Comment by felixfontein** [comment]
@phumpal can you take a look at this PR? Thanks :)

**Comment by phumpal** [comment]
Well, hello again @felixfontein.

I very much approve :+1:. Looks good, thanks @leedm777

File: plugins/modules/monitoring/airbrake_deployment.py (hunk 3 above)
Comment (leedm777) [reply plugins/modules/monitoring/airbrake_deployment.py]: Cool. I've got the branch ready to submit a new PR once this one is merged.

**Comment by felixfontein** [comment]
@leedm777 thanks a lot for implementing this!
@phumpal thanks a lot for reviewing this!

**Pull request merged**